server/routes/documents/letters-list/letters-list.controller.js

```diff
@@ -382,4 +409,12 @@
     }).length;
   }
 
+  function areAllChecked(req) {
+    if (!req.session.documentsJurorsList.checkedJurors) {
+      req.session.documentsJurorsList.checkedJurors = [];
+    }
+    return req.session.documentsJurorsList.data.filter((juror) => (!juror[4] && !juror[5])).length
+      === req.session.documentsJurorsList.checkedJurors.length;
```

Comment:
the `!juror[4]` and `!juror[5]` should also probably be inverted

Reply:
Updated